internal/exec/tar_utils.go

```diff
@@ -0,0 +1,92 @@
+package exec
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	u "github.com/cloudposse/atmos/pkg/utils"
+)
+
+// extractTarball extracts the tarball file into the destination directory
+func extractTarball(sourceFile, extractPath string) error {
+	file, err := os.Open(sourceFile)
+	if err != nil {
+		return err
+	}
+
+	defer func(file *os.File) {
+		err := file.Close()
+		if err != nil {
+			u.LogError(fmt.Errorf("\nerror closing the file '%s': %v\n", sourceFile, err))
```

Comment:
It seems wrong that we are adding new lines to the log messages. I know why it's needed, such as to clear the line, but it seems like `u.LogError` should handle that and not the user.
```suggestion
			u.LogError(fmt.Errorf("\nerror closing the file '%s': %v\n", sourceFile, err))
```